app/javascript/components/shared/helpers.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export const preserveLines = string => `${string}`.split("\n").map((line, idx) => <p key={idx}>{line}</p>);
```

Comment:
lets use String(string) if we need here to cast this into string instead of empty interpolation